packages/onboarding-ui/src/pages/OrganizationInfoPage/OrganizationInfoPage.tsx

```diff
@@ -16,13 +18,15 @@ type OrganizationInfoPageProps = {
   initialValues?: OrganizationInfoPayload;
   onSubmit: SubmitHandler<OrganizationInfoPayload>;
   onBackButtonClick: () => void;
+  onConfirmText?: string;
```

Comment:
Better keep `on` prefixes for event callback props?

Reply:
Sounds better, thnks!